### PR TITLE
Add electron menubar support

### PR DESCRIPTION
**Pull request by kwbauson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "isbenmarriedyet.com",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/jpetrucciani/isbenmarriedyet.com.git",
+  "license": "MIT",
+  "scripts": {
+    "start": "electron app.js"
+  },
+  "dependencies": {
+    "electron": "^8.3.0",
```
Comment (cgaeta) [inline package.json:11]: electron adds too many dependencies, downvoted

**Comment by kwbauson** [comment]
Plz merge thx

**Comment by kwbauson** [comment]
What's blocking this right now? How can we get it merged?

**Comment by jpetrucciani** [comment]
@kwbauson does it work?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Build/release
+
+on: push
```
Comment (jpetrucciani) [inline .github/workflows/release.yml:3]:
```yaml
on:
  push:
    branches: [master]
```

**Pull request closed without merging**